### PR TITLE
Added documentation for tracing default resolves for Apollo

### PR DESCRIPTION
**Pull request by nilansaha**
Added docs for this [PR](https://github.com/mirumee/ariadne/pull/608) as asked [here](https://github.com/mirumee/ariadne/pull/608#issuecomment-1036371741)

### CODE REVIEW

File: docs/apollo-tracing.md [hunk 1]
```diff
@@ -23,3 +23,17 @@ app = GraphQL(
 ```
 
 > **Note:** If you are using WSGI, use `ApolloTracingExtensionSync` in place of `ApolloTracingExtension`.
+
+## Tracing default resolvers for Apollo
```
Comment (rafalp) [inline docs/apollo-tracing.md:27]:
```suggestion

## Tracing default resolvers
```

File: docs/apollo-tracing.md [hunk 2]
```diff
@@ -23,3 +23,17 @@ app = GraphQL(
...
+
+By default tracing of default resolvers are set to False but if you want to trace base field types for which resolvers are not explicitly set then it can be done by setting the parameter `trace_default_resolver` to `True`. This is useful when used in conjunction with Apollo Studio to see which base fields are being accessed for a entity and thereby helping api lifecycle management.
```
Comment (rafalp) [inline docs/apollo-tracing.md:29]:
```suggestion
For performance reasons tracing is disabled for default resolvers. To reverse this behavior you can initialize `ApolloTracingExtension` with `trace_default_resolver` option set to `True`:
```

**Review by rafalp — CHANGES_REQUESTED** [review]

**Comment by nilansaha** [comment]
Made changes

**Review by rafalp — APPROVED** [review]

**Comment by rafalp** [comment]
Thank you!

**Pull request merged**